### DISCUSSION TITLE
Clearing store frames for disabled cameras

### DISCUSSION THREAD

**Discussion by brubaked** [blakeblackshear/frigate#4245] [Ask A Question]
Hi, 
I have a number of cameras that I periodically turn off. After turning off the camera, frigate continues to show the last recieved frame. Is there a way to clear that frame so it just shows a green screen when the camera is turned off?

Thanks!

**Comment by NickM-27** [comment]
Currently no, there is no way to do that

**Reply by brubaked** [reply]
Thanks for verifying that. If anyone comes across this later and wants to do the same thing, I figured out a very hacky, but functional workflow to accomplish this. 

After disabling the cameras, you can just restart frigate. I've just tested it with the home assistant addon and it seems to give the desired result.

**Comment by kilbee** [comment]
Isn't that a bit problematic? For example when camera gets disconnected and you don't really notice until checked very closely?

**Reply by NickM-27** [reply]
In the context of not having other options, sure. When there's other options like notifications or UI to show a camera is not currently connected then I don't think it is a problem.

**Reply by brubaked** [reply]
Does the main frigate UI have an indication that a camera is not reporting? Besides the viewport being green due to me restarting the addon, I don't think there's an indication. 

![Screenshot_20221111_113625](https://user-images.githubusercontent.com/37672083/201417026-e1da073d-7ae9-4548-926c-5d18093a3bf6.jpg)

**Reply by NickM-27** [reply]
On the debug page there is the fps which will be 0, if you open the camera the stream won't start, and yes the green screen. 

In the next update we're looking to change the green screen to something more obvious

**Reply by kilbee** [reply]
This is a good tip actually, for me - frigate used with HomeAssistant - this is more than enough to detect when camera is offline.
